src/mocks/mock-request.ts

```diff
@@ -32,24 +32,24 @@ export const mockInitActions = (): Promise<void> => {
  * Example of an action performed by the Session ednpoint
  * @param {string} sessionId - ID of the current session
  */
-export const mockInitResources = (sessionId: string): Promise<any[]> => {
+export const mockInitResources = (sessionId: string): Promise<Memory[]> => {
+  
+  const memoryData1 : Memory = {
```

Comment:
Could we remove the extra space please?
```suggestion
  const memoryData1: Memory = {
```